landscape/lib/schema.py

```diff
@@ -1,4 +1,5 @@
 """A schema system. Yes. Another one!"""
+import six
```

Comment:
Seems like six is a dependency of landscape server. I don't see used anywhere in client. I'm not totally sure if six is pre-installed in all places where we can run landscape cilent